### PR TITLE
Transfer prompt improvements (block list warning + memo handling + i18n fixes)

### PR DESCRIPTION
**Pull request by grctest**
Fix bitsharesjs reference
Show memo in transfer prompt if it exists

image

### CODE REVIEW

**Comment by grctest** [comment]
If the user is prompted to transfer assets on the bitshares blockchain to an account which is blocked by `committee-blacklist-manager` they will be shown a warning now:

image

**Comment by abitmore** [comment]
@grctest thanks for the efforts. If you don't already know, please check https://github.com/bitshares/bitshares-ui/pull/3454#issuecomment-1053463753 for potential issues related to memo encryption.

**Comment by abitmore** [comment]
BTW @sschiessl-bcp made some efforts about memo in https://github.com/bitshares/beet/pull/171, perhaps it helps.

**Comment by abitmore** [comment]
I think #171 is a bit outdated. Please check https://github.com/bitshares/bitshares-ui/pull/3454#issuecomment-1053463753.

**Comment by grctest** [comment]
Alright, I've introduced the bitshares-ui implementation for memo encryption, this allows for unencrypted memo as well as the optional nonce value.

**Comment by grctest** [comment]
This PR has been further extended to support changing i18n locales, you're now able to switch between English & German with a button click in the headers.

**Comment by grctest** [comment]
The i18n component was added to prompts 👍

File: src/components/popups/transferrequestpopup.vue [hunk 1]
```diff
@@ -140,6 +181,15 @@
                 </ui-item-text-content>
             </ui-item>
         </ui-list>
+        
+        <ui-alert v-if="warning" state="warning">
+            {{
+                warning === "serverError"
+                    ? "Couldn't check if account is safe, proceed with caution."
```
Comment (sschiessl-bcp) [inline src/components/popups/transferrequestpopup.vue:188]: If a user sees this he could think that accounts that do not have the warning are safe, this delivers way too much safety feeling

Comment (grctest) [reply src/components/popups/transferrequestpopup.vue]: Do you want a default warning of caution? Or do you have a proposal for an alternative string?

**Pull request merged**